src/signals/incident/components/form/CategorySelect/CategorySelect.js

```diff
@@ -1,19 +1,42 @@
-import React, { useMemo, useCallback } from 'react';
+import React, { useMemo, useCallback, useState, useEffect } from 'react';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
 
 import Select from 'components/SelectInput';
 import { useSelector } from 'react-redux';
 import { makeSelectSubCategories } from 'models/categories/selectors';
+import InfoText from 'components/InfoText';
+
+const Wrapper = styled.div`
+  width: 100%;
+  margin-bottom: 0;
+  p {
```

Comment:
How do you know that a `p` tag is rendered as a child of `Wrapper`? Looking at the code of the `CategorySelect` component, that isn't obvious.

Reply:
I moved the styling to the InfoText component